components/serializers.py

```diff
@@ -110,25 +109,26 @@ def collect_catalog_data(controls: list, catalog_versions: List[Catalog.Version]
 
 
 def collect_component_data(component: dict) -> dict:
-    tools = ComponentTools(component)
-
-    control_list = tools.get_controls()
-    controls: dict = {}
-    for control in control_list:
-        controls[control.get("control-id")] = {
-            "narrative": control.get("description"),
-            "responsibility": get_control_responsibility(control, "security_control_type"),
-            "provider": get_control_responsibility(control, "provider"),
-        }
-
-    component_ = tools.get_components()[0]
+    component_model = Model(**component)
+    component_def = component_model.component_definition.components[0]
 
     return {
-        "title": component_.get("title"),
-        "description": component_.get("description"),
-        "standard": component_.get("control-implementations")[0].get("description"),
-        "source": component_.get("control-implementations")[0].get("source"),
-        "controls": controls
+        "title": component_def.title,
+        "description": component_def.description,
+        "standards": {
+            item.description: {
+                "source": item.source,
+                "controls": {
+                    control.control_id: {
+                        "narrative": control.description,
+                        "responsibility": control.responsibility,
+                        "provider": control.provider
+                    }
+                    for control in item.implemented_requirements
+                }
+            }
```

Comment:
Thank you for this- the similar structure to the catalog data that comes back makes it much easier to understand when working with the data